holoviews/plotting/bokeh/element.py

```diff
@@ -104,6 +103,12 @@ class ElementPlot(BokehPlot, GenericElementPlot):
           * timeout   - Timeout (in ms) for checking whether interactive
                         tool events are still occurring.""")
 
+    sizing_mode = param.ObjectSelector(default='fixed',
+                                       objects=[
+            'fixed', 'scale_width', 'scale_height',
+            'scale_both', 'stretch_both'
+        ], doc="""Defines how the plot scales when resized.""")
+
```

Comment:
What it usually means in the projects I'm familiar with is *definitely* not ready to merge, *probably* not ready for heavyhanded, very detailed comments, and *probably* ready for someone to take a quick look and start giving some feedback.  If that's not the case for a given WIP PR, just declare it in text so that other humans know what to do with it.  No big deal...